src/Utilities/FlowAnalysis/FlowAnalysis/Analysis/TaintedDataAnalysis/WebInputSources.cs

```diff
@@ -19,6 +21,21 @@ static WebInputSources()
         {
             var sourceInfosBuilder = PooledHashSet<SourceInfo>.GetInstance();
 
+            sourceInfosBuilder.AddSourceInfo(
+                WellKnownTypeNames.MicrosoftAspNetCoreMvcControllerBase,
+                 new ParameterMatcher[]{
+                    (parameter) => {
+                        ISymbol containingSymbol = parameter.ContainingSymbol;
+
+                        if (containingSymbol.DeclaredAccessibility != Accessibility.Public)
+                            return false;
+
+                        if (containingSymbol.IsConstructor())
```

Comment:
(Then this `if` could be removed)

Reply:
Do you mean `return !containingSymbol.IsConstructor();`?

Reply:
Ah, I see
```cs
if (containingSymbol.DeclaredAccessibility != Accessibility.Public
                            || containingSymbol.IsConstructor())
```